### PR TITLE
Add more documentation on virtualenvs and install to the quickstart.

### PR DESCRIPTION
**Pull request by Jason94**
@shaunagm and I have been talking about adding more documentation for installing Parsons and making virtual environments, particularly for Windows. This PR adds a "virtual environments" section and fleshes out the installation section to the [index page of the documentation](https://move-coop.github.io/parsons/html/index.html).

I have only used Parsons on Windows, so I haven't added virtual environment instructions for Linux/Mac OS. If anyone would like to volunteer, I'd be happy to merge some changes into this branch before we merge the PR. Otherwise we could merge this PR and add that section later.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -71,9 +71,68 @@ Sources
 * Documentation: `<https://move-coop.github.io/parsons/html/index.html>`_
 * Source Code: `<https://github.com/move-coop/parsons>`_
 
+Virtual Environments
+====================
+
+Normally, tools like `pip` install Python libraries directly to your system. Python scripts
+or libraries look for their dependencies in your system. This can cause problems when you
+have two scripts/libraries installed that require differnt versions of the same library.
```
Comment (neverett) [inline docs/index.rst:79]: "differnt" -> "different"

File: docs/index.rst [hunk 2]
```diff
@@ -71,9 +71,68 @@ Sources
...
+
+      git clone git://github.com/davidmarble/virtualenvwrapper-win.git
+      cd virtualenvwrapper-win
+      python setup.py install
+
+2. Find the ``Scripts\`` directory for your Python installation, such as ``C:\Users\<User>\AppData\Local\Programs\Python\Python37\Scripts\``.
+
+3. Add the ``Scripts\`` directory to your Path.
+
+4. To create a virtual environment for Parsons, execute this command: ``mkvirtualenv parsons``
```
Comment (neverett) [inline docs/index.rst:100]: Very minor, but for consistency, I'd recommend structuring this and the following step the same way, maybe something like:

"To create a virtual environment for Parsons, execute: ``mkvirtualenv parsons`` 
"To use this virtual environment, execute: ``workon parsons``"

File: docs/index.rst [hunk 3]
```diff
@@ -71,9 +71,68 @@ Sources
...
+Linux / Mac OS
+------------------------------
+
+<Coming Soon>
+
 Installation
 ============
-You can install the most recent release by running: ``pip install parsons``
+
+There are two methods of installing Parsons: Using pip, and from source. Use pip if you just want to install Parsons and start using it. Install from source if you might want to patch Parsons to customize its behavior.
```
Comment (neverett) [inline docs/index.rst:113]: Another minor edit, but for a slightly smoother sentence, maybe "There are two ways to install Parsons..."

File: docs/index.rst [hunk 4]
```diff
@@ -71,9 +71,68 @@ Sources
...
+
+Installing Parsons (pip)
+-----------------------------
+
+1. Make sure you're in your parsons virtual environment.
```
Comment (neverett) [inline docs/index.rst:118]: Suggestion: "Make sure you're in your Parsons virtual environment: ``workon parsons``"

File: docs/index.rst [hunk 5]
```diff
@@ -71,9 +71,68 @@ Sources
...
+
+2. Execute: ``pip install parsons``
+
+Installing Parsons from Source
+----------------------------------
+
+
+1. Make sure you're in your parsons virtual environment.
```
Comment (neverett) [inline docs/index.rst:126]: See above suggestion under "Installing Parsons (pip)"

File: docs/index.rst [hunk 6]
```diff
@@ -71,9 +71,68 @@ Sources
...
+
+2. Execute these commands::
```
Comment (neverett) [inline docs/index.rst:128]: Remove second colon

File: docs/index.rst [hunk 7]
```diff
@@ -71,9 +71,68 @@ Sources
...
+
+      git clone https://github.com/move-coop/parsons.git
+      pip install -r requirements.txt
+      python setup.py install
+
+
+3. To update your installation, pull the most recent branch from the Parsons GitHub repo and rerun ``python setup.py install``.
```
Comment (neverett) [inline docs/index.rst:135]: I would include the command for pulling the most recent branch from the GitHub repo (`git pull origin master`) for clarity

**Review by neverett — APPROVED** [review]
I left some suggested edits, but otherwise, this looks good to me, thank you for adding this documentation!

File: docs/index.rst (hunk 6 above)
Comment (Jason94) [reply docs/index.rst:128]: The double colons are for the `.rst` command-block syntax. Only one colon shows up in the compiled HTML file.

**Comment by Jason94** [comment]
> I left some suggested edits, but otherwise, this looks good to me, thank you for adding this documentation!

Thanks for the review! I've made the changes in a second commit.

**Review by neverett — APPROVED** [review]
Looks good to me!

**Pull request merged**